src/wasm-binary.h

```diff
@@ -331,10 +331,7 @@ enum Section {
   Event = 13
 };
 
-enum SegmentFlag {
-  IsPassive = 0x01,
-  HasMemIndex = 0x02,
-};
+enum SegmentFlag { IsPassive = 0x01, HasMemIndex = 0x02, Is64 = 0x04 };
```

Comment:
Does clang-format really prefer this?  Yuck.  Who am I to argue though.

Reply:
We can change it by adding `AllowShortEnumsOnASingleLine: false` in our `.clang-format` if you want.. It will also require changing other `enum`s though (there's not _that_ much):
```
aheejin@aheejin:~/binaryen/src$ grw enum | grep { | grep ,
abi/js.h:enum class LegalizationLevel { Full = 0, Minimal = 1 };
cfg/liveness-traversal.h:  enum What { Get = 0, Set = 1, Other = 2 };
cfg/Relooper.h:  enum ShapeType { Simple, Multiple, Loop };
emscripten-optimizer/parser.h:  enum Type { Binary = 0, Prefix = 1, Postfix = 2, Tertiary = 3 };
ir/module-utils.h:  enum IndirectCalls { IgnoreIndirectCalls, IndirectCallsHaveProperty };
ir/trapping.h:enum class TrapMode { Allow, Clamp, JS };
passes/RemoveUnusedModuleElements.cpp:enum class ModuleElementKind { Function, Global, Event };
passes/DeadArgumentElimination.cpp:  enum LocalUse { Read, Written };
passes/Asyncify.cpp:enum class State { Normal = 0, Unwinding = 1, Rewinding = 2 };
passes/Asyncify.cpp:enum class DataOffset { BStackPos = 0, BStackEnd = 4 };
support/threads.h:enum class ThreadWorkState { More, Finished };
support/command-line.h:  enum class Arguments { Zero, One, N, Optional };
support/file.h:enum BinaryOption { Binary, Text };
wasm/wasm-validator.cpp:    enum { UnsetArity = Index(-1), PoisonArity = Index(-2) };
wasm/literal.cpp:enum class LaneOrder { Low, High };
wasm-binary.h:enum Meta { Magic = 0x6d736100, Version = 0x01 };
wasm-binary.h:enum MemoryFlags { HasMaximum = 1 << 0, IsShared = 1 << 1 };
wasm-builder.h:  enum Mutability { Mutable, Immutable };
wasm.h:enum class IRProfile { Normal, Poppy };
wasm.h:enum HostOp { MemorySize, MemoryGrow };
wasm.h:enum AtomicRMWOp { Add, Sub, And, Or, Xor, Xchg };
wasm.h:enum SIMDTernaryOp { Bitselect, QFMAF32x4, QFMSF32x4, QFMAF64x2, QFMSF64x2 };

aheejin@aheejin:~/binaryen/src$ grw enum | grep { | grep , | wc -l
22
```

Reply:
Yes, that sounds like a good flag to set.. independent from this PR :)